.github/workflows/ci-pypi-publish-README.md

```diff
@@ -0,0 +1,201 @@
+# 🐍 Reusable PyPI Publish workflow
+
+This reusable workflow is part of the City of Helsinki's GitHub Actions setup, specifically designed to provide an opinionated and consistent PyPI package publishing process for City of Helsinki's Python projects.
+
+## 🌟 Key Features
+
+- **Flexible Python Versions**: Configurable Python version support.
+- **Automated Testing**: Runs project tests before publishing (can be skipped if needed).
+- **Build Process**: Builds Python packages using the standard `build` module.
+- **Dual Authentication**: Supports both API tokens and trusted publishers for secure publishing.
+- **Dependency Management**: Optional requirements file installation for complex dependencies.
+- **Modern Tools**: Uses latest Python setup and build tools.
+
+## 📋 Requirements for Projects Using the Workflow
+
+- **pyproject.toml** or **setup.py** for package configuration.
+- **Source code** in a standard Python package structure.
+- **Test suite** (pytest recommended) if running tests.
```

Comment:
Slighlty redundant and also incorrect; if you run tests you then yes, you probably want to have some tests to run with it, but you can have a test suite with zero tests.